frontend/coprs_frontend/commands/remove_warning_banner.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+import click
+
+from coprs.constants import BANNER_LOCATION
+
+
+@click.command()
+def remove_warning_banner() -> None:
```

Comment:
I would prefer merging this with `add_warning_banner` and simply having `warning_banner` with an optional parameter `--remove`. Or at least having these two functions in the same file.